src/DataProtection/DataProtection/src/Internal/ContainerUtils.cs

```diff
@@ -101,9 +102,18 @@ private static bool IsProcessRunningInContainer()
                 return false;
             }
 
-            var lines = File.ReadAllLines(procFile);
-            // typically the last line in the file is "1:name=openrc:/docker"
```

Comment:
Please keep the comment

Reply:
@HaoK, I thought that comment was an attempt to justify the use of `Enumerable.Reverse()`.

But the file lines are always read from start to end.

I fail to understand the significance of the comment.